_layouts/base.html

```diff
@@ -3,17 +3,16 @@
 <!--
     ============================================================================
 
-    Hey! If you're viewing this, you should probably come work on our Technology
-    & Innovation team. We're always looking for a few great designers,
-    developers, data scientists, and network, infrastructure, privacy and
-    security pros. Keep an eye on our job opportunities at:
-    http://www.consumerfinance.gov/jobs/
+    Hey! If you're viewing this, you should probably come work on our
+    Technology & Innovation team. We're always looking for a few great
+    designers; developers; data scientists; and network, infrastructure,
```

Comment:
Because the series breaks down like this:

1. designers
2. developers
3. data scientests
4. network, infrastructure, privacy, and security pros

Because the last item in the series it itself a series that requires commas in its construction, the items in the parent series should be separated by semicolons.